src/python/CMSMonitoring/StompAMQ.py

```diff
@@ -202,7 +207,8 @@ def _send_single(self, conn, notification):
         return
 
     def make_notification(self, payload, docType, docId=None, producer=None, ts=None, metadata=None,
-                          dataSubfield="data", schema=None, returnValidationResult=True):
+                          dataSubfield="data", schema=None, returnOffendingKeys=False,
```

Comment:
I think we agreed to change default behavior and use schema as mandatory parameter. If users want to skip it they should explicitly provide schema=None. But since schema will be mandatory parameter it should come before optional parameter, i.e. be moved before docId.

Reply:
That's a different place in the code. The required parameter is in the constructor of StompAMQ. Here in `make_notification` the default behavior (i.e. giving `None`) is to fall back to the schema provided when setting up StompAMQ. I think it's ok as it is.

But highlighting this made me notice that I forgot to remove the `returnOffendingKeys` parameter from the function declaration. I'll add another commit to fix it.